frontend/apps/reader/modules/readergesture.lua

```diff
@@ -631,6 +631,36 @@ Default value: %1]]), GestureDetector.PAN_DELAYED_INTERVAL/1000),
                     UIManager:show(items)
                 end,
             },
+            {
```

Comment:
to add this to the redone gestures plugin it should be put here:
https://github.com/koreader/koreader/blob/9e0ac734baf704f708c19150f41442ead7745c09/plugins/gestures.koplugin/main.lua#L538